docker/quickstart/docker-compose.yml

```diff
@@ -249,8 +251,6 @@ services:
     depends_on:
       - kafka-setup
       - datahub-gms
-    command: "sh -c 'while ping -c1 kafka-setup &>/dev/null; do echo waiting for kafka-setup... && sleep 1; done; \
```

Comment:
Somehow  the command got dropped after you reverted your change?

Reply:
@mars-lan This command is not necessary anymore as I have added an entrypoint in `docker/mce-consumer/Dockerfile`

Reply:
Actually, "entrypoint" doesn't totally replace this command. This command provides the orchestration between Kafka & MCE consumer containers. We need to make sure that MCE consumer starts after kafka-setup is done, otherwise it will fail because MetadataChangeEvent topic is not generated on Kafka yet.

Reply:
Makes sense! Pushed updates.